.github/workflows/run-deploy.yaml

```diff
@@ -110,12 +110,16 @@ jobs:
         name: Get Lightsail Instance IP Address
         run: echo ::set-output name=IP_ADDRESS::$(aws lightsail get-instance-access-details --instance-name project-lab-${{ steps.get-branch.outputs.branch }} | jq -r '.accessDetails.ipAddress')
 
+      - id: get-lightsail-ssh-key
+        name: Get Lightsail Private SSH Key
+        run: mkdir ~/.ssh && aws lightsail download-default-key-pair | jq '.privateKeyBase64' | tr -d '"' | sed 's/\\n/\n/g' > ~/.ssh/lightsail.pem
```

Comment:
Playing a little bit with the second command (the `aws lightsail...`), I realized it can be shortened even more:

```sh
aws lightsail download-default-key-pair | jq -r '.privateKeyBase64' | head -n -1
```

The `-r` flag in `jq` will output the result of the filter as raw text instead of as a JSON string; that gets rid of the need for `tr` and `sed`.

`head -n -1` will discard the last line, which is just an empty line and is not needed.